lib/player.rb

```diff
@@ -0,0 +1,79 @@
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :total, :highest_scoring_word, :highest_word_score, :tiles
+
+    def initialize(input_name)
+      @name = input_name
+      @plays = []
+      @total = 0
+      @highest_word_score = 0
+      @highest_scoring_word = "marisa"
+      @tiles = []
+    end
+
+    def play(input_word)
+      total_score #TODO 99 vs. 101 edge case.
+
+      if won? == true
+        return false
+      end
+
+      # check_tiles(input_word) # => OPTIONAL 1
+
+      score = Scrabble::Scoring.score(input_word)
+      @plays << input_word
+      total_score
+
+      return score
+    end
+
+    def total_score
+      return 0 if @plays.length == 0
+      scores = @plays.map{ |word| Scrabble::Scoring.score(word) }
+      @total = scores.inject(:+)
```

Comment:
instead of recalculating the scores of each word in plays, there may be opportunity to refactor this